src/System.Net.Security/src/System/Net/NTAuthentication.cs

```diff
@@ -523,13 +428,11 @@ internal byte[] GetOutgoingBlob(byte[] incomingBlob, bool throwOnError, out Inte
                 SSPIHandleCache.CacheCredential(_credentialsHandle);
             }
 
-            // The return value from SSPI will tell us correctly if the
-            // handshake is over or not: http://msdn.microsoft.com/library/psdk/secspi/sspiref_67p0.htm
-            // we also have to consider the case in which SSPI formed a new context, in this case we're done as well.
-            if (statusCode == Interop.SecurityStatus.OK)
+            // The return value will tell us correctly if the handshake is over or not
+            if (statusCode == SecurityStatusPal.OK)
             {
                 // Success.
-                if (statusCode != Interop.SecurityStatus.OK)
+                if (statusCode != SecurityStatusPal.OK)
```

Comment:
I'm confused by this line.  We just verified that statusCode == SecurityStatusPal.OK... isn't this dead code?